modules/local/gstama/filelist/main.nf

```diff
@@ -0,0 +1,33 @@
+process GSTAMA_FILELIST {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img' :
```

Comment:
This image was causing some issues on GCP and it was decided to change it to the base Ubuntu one, see [here](https://github.com/nf-core/rnaseq/issues/764).  So You will need to replace any containers for `biocontainers/biocontainers:v1.2.0_cv1` with `ubuntu:20.04` and example [here](https://github.com/drpatelh/nf-core-viralrecon/blob/870bd0170a84dc28439520716aac51770b0bb506/modules/nf-core/modules/cat/fastq/main.nf#L6-L8)

Reply:
Modifications applied